pkg/apply/task/apply_task_test.go

```diff
@@ -114,7 +114,7 @@ func TestApplyTask_BasicAppliedObjects(t *testing.T) {
 			require.NoError(t, err)
 
 			actual := taskContext.AppliedResources()
-			if !object.SetEquals(expected, actual) {
+			if !object.IsSetEqual(expected, actual) {
```

Comment:
Remove this change. If you want to advocate for a name change, please do it in another PR.